src/main/java/net/sf/jabref/gui/preftabs/AppearancePrefsTab.java

```diff
@@ -182,7 +182,7 @@ public AppearancePrefsTab(JabRefPreferences prefs) {
         overrideFonts.addActionListener(e -> fontSize.setEnabled(overrideFonts.isSelected()));
 
         fontButton.addActionListener(
-                e -> new FontSelectorDialog(null, GUIGlobals.currentFont).getSelectedFont().ifPresent(x -> usedFont = x));
+                e -> new FontSelectorDialog(null, usedFont).getSelectedFont().ifPresent(x -> usedFont = x));
```

Comment:
It works, when the variable usedFont is a field: http://stackoverflow.com/questions/25055392/lambdas-local-variables-need-final-instance-variables-dont